dist/js/keditor.js

```diff
@@ -1521,14 +1521,11 @@
                 }
                 
                 var settingComponent = self.getSettingComponent();
-                if (settingComponent) {
+                if (settingComponent.length > 0) {
```

Comment:
@sepehr09 You check `settingComponent && settingComponent.length > 0` to void case `settingComponent` is `null`.